wolfscript.js

```diff
@@ -130,6 +130,20 @@ function wolfKaren() {
     img.src = "girls/karen.png";
 }
 
+function loadWolf(character) {
```

Comment:
This function is called by onload and onhashchange. We can probably reduce conditional complexity by introducing an MVC* pattern (as seen in AngularJS).